backend/drive/drive.go

```diff
@@ -1237,28 +1234,35 @@ func (f *Fs) dirchangeNotifyRunner(notifyFunc func(string), pollInterval time.Du
 			return
 		}
 
-		pathsToClear := make([]string, 0)
+		pathsToClear := make(map[string]fs.EntryType)
```

Comment:
Using a map instead of a slice has the potential to get them out of order.  Do you think this will matter?  If we want to preserve the order then easy enough to use `[]struct{string, fs.EntryType}`

Reply:
I don't believe it matters. The previous implementation was sorting them anyway: `sort.Strings(pathsToClear)` so that broke the order it got them from the cloud provider.

Reply:
I guess I was thinking ahead if we were to be slightly more specific in the notifications - this file deleted, this directory renamed for which the order would be more important.

Reply:
@ncw Fair enough. That's a good point. I switched to an array of struct to keep the order and adjusted the test to verify that the order is kept too.